docs/deployment-design-aws.md

```diff
@@ -0,0 +1,189 @@
+# Recommended design for DCL MainNet deployment on AWS
+## AWS deployment diagram
+![AWS deployment diagram](./deployment-aws.png)
+
+## Recommended IaC frameworks
```

Comment:
Let's mention, that we are going to support a modular approach:
- Terraform - set up environment of one or multiple types: 
  - Validator
  - Private Sentries
  - Public Sentries
  - Observers
  - Load Balancers.
- Ansible - provision of the following node types:
  - Genesis Validator
  - Non-genesis validator
  - private sentry
  - public sentry
  - observer 
  
Also let's mention what option will be used for provisioning of new nodes into an existing network. Is it state sync, or copy-paste of data from existing nodes, or anything else.